src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
```

Comment:
[CODE STYLE] - Avoid variable namings like `array`, `object` `arr`, `obj`, `result`. It will decrease code readability. However, in this snippet, variable naming is appropriate and clear.